vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridOrderColumnsIT.java

```diff
@@ -64,6 +64,14 @@ public void gridOrder_321_123() {
         assertColumnHeaders("Col1", "Col2", "Col3");
     }
 
+    @Test
+    public void gridVisualOrderReset() {
```

Comment:
I have no idea based on the test name what is the case that is actually covered.
I'd prefer something like
`clientReorderColumns_serverResetOrder`

Not blocking.

Reply:
Done.